docs/responsive-styles.md

```diff
@@ -79,4 +79,4 @@ export default {
 <Box width={{ _: 1, sm: 1, md: 1 / 2, lg: 1 / 4 }} />
 ```
 
-Read more in the [Array Props Guide](/guides/array-props).
```

Comment:
Thanks, but this is intended to link on the site https://styled-system.com – currently there isn't a mechanism in place to convert `.md` links to routes in the docs site, but if you'd like to open a PR for that, feel free to do so